source/documentation/monitoring-alerts/bind-exporter-to-prometheus.md

```diff
@@ -21,7 +27,7 @@ service          plans        description
 gds-prometheus   prometheus   GDS internal Prometheus monitoring alpha https://reliability-engineering.cloudapps.digital/#metrics
 ```
 
-If you're unable to see `gds-prometheus` in the output of `cf marketplace` please contact us through the [#re-prometheus-support Slack channel].
+If you're unable to see `gds-prometheus` in the output of `cf marketplace` please contact us through the [#re-prometheus-support Slack channel] - it's likely your org hasn't been granted access to the `gds-prometheus` service.
```

Comment:
Not a blocker for this PR but I wonder if we should point people towards #paas for this

Reply:
:man_shrugging: Historically this process lived with autom8 (and observe before them)...